### PR TITLE
adding override data to key drivers bucket

### PR DESCRIPTION
**Pull request by bsdangra**

### CODE REVIEW

File: soiapis/utils.py [hunk 1]
```diff
@@ -71,6 +71,16 @@ def get_output_data(model_name):
 
     return op_section_data_template, override_section
 
+def get_override_buc_data(model_name):
+    output_data = OutputSectionData.objects.filter(model_name=model_name)
```
Comment (nabendukarmakar) [inline soiapis/utils.py]: why don't we query with `level_name` as well?

File: soiapis/views.py [hunk 2]
```diff
@@ -308,10 +308,12 @@ def get_base_kbd_data(model_name, startMonthDate, endMonthDate, startMonthDateYA
         j = i.replace("planned", "actual")
         if due_to_planned[i] and due_to_planned[i] > 0 and due_to_actual_ya[j] and due_to_actual_ya[j] > 0:
             due_to_planned_contribution[i] = (due_to_planned[i]/due_to_actual_ya[j])-1
-    
+
 
     if kbd_data_json:
         output_data = get_output_data(model_name)
+        #override buc data
+        override_buc_data = get_override_buc_data(model_name)
```
Comment (nabendukarmakar) [inline soiapis/views.py]: `get_output_data` also makes a DB call and `get_override_buc_data` also now.
Can we optimize this?
Why dont' we call `get_override_buc_data` inside `get_output_data` and pass on the model_obj to save on database call?

File: pgdjangosql/templates/src/utils/analyzer-section-utils.js [hunk 3]
```diff
@@ -212,11 +210,7 @@ export function getKbdDriversCalculation(
       "channel_output_data"
     ] = channelWiseOutputData(test_obj, startDtYA, endDtYA);
 
-    let channelWiseOfftakeData = channelWiseOutputData(
-      test_obj,
-      startDtYA,
-      endDtYA
-    );
+    let channelWiseOfftakeData = channelWiseOutputData(test_obj,startDtYA,endDtYA);
```
[the remark this answers is not in the document]
Comment (bsdangra) [reply pgdjangosql/templates/src/utils/analyzer-section-utils.js]: Changed it to "output_section_obj"

File: soiapis/utils.py (hunk 1 above)
Comment (bsdangra) [reply soiapis/utils.py]: Filtering it with level_name as well

File: soiapis/views.py (hunk 2 above)
Comment (bsdangra) [reply soiapis/views.py]: Now filtering it with level_name

**Comment by nabendukarmakar** [comment]
Looks fine!

**Pull request merged**